data/co/people/Mary-Young-ffc5caa4-29c4-475a-b662-a814bae9f8c5.yml

```diff
@@ -0,0 +1,16 @@
+id: ocd-person/ffc5caa4-29c4-475a-b662-a814bae9f8c5
+name: "\u2018Mary\u2019 \u2018Young\u2019"
```

Comment:
same here, the single quotes you presumably passed in on the command line are being reflected here, depending on your shell you'll want to use double quotes (you can also omit them if the string is something simple like Mary)

Reply:
Resolved.